src/env.d.ts

```diff
@@ -9,6 +9,7 @@ declare const webpackEnv: {
 declare module NodeJS {
     interface ProcessEnv {
         NODE_ENV: 'development' | 'production' | 'test'
+        storybook?: boolean
```

Comment:
Why not add 'storybook' into `NODE_ENV`?

Reply:
storybook itself can build in dev mode or prod mode

Reply:
Uppercase is better.